jujugui/static/gui/src/app/components/profile/credential-list/_credential-list.scss

```diff
@@ -124,6 +124,25 @@
   .dropdown-menu__list-item-link {
     padding: 0 0 0.5rem;
   }
+
+  // XXX These override styles are required until the <PopUp> components is converted to Vanilla v1
+  .button-row {
+    width: 100%;
+    margin: 0;
+    padding: 0;
+    display: grid;
```

Comment:
Why grid instead of flex?

Reply:
No particular reason - I think it's six of one, half a dozen of the other in this case.